contents/docs/libraries/ios/index.mdx

```diff
@@ -378,6 +378,34 @@ To set up [session replay](/docs/session-replay/mobile) in your project, all you
 
 [Surveys](/docs/surveys) launched with [popover presentation](/docs/surveys/creating-surveys#presentation) are automatically shown to users matching the [display conditions](/docs/surveys/creating-surveys#display-conditions) you set up.
 
+## Method swizzling
+
+The PostHog iOS SDK uses method swizzling to intercept and modify method calls at runtime to provide advanced features like:
+
+- **Screen view tracking** - Automatically captures when view controllers are presented (`config.captureScreenViews`)
+- **Element interactions** - Automatically tracks user interactions with UI elements (`config.captureElementInteractions`)  
+- **Session replay** - Records user sessions (`config.sessionReplay`)
+- **Surveys** - Displays surveys at appropriate times (`config.surveys`)
+- **Advanced session management** - Provides more precise session ID calculation and rotation by detecting user activity and idleness
```

Comment:
```suggestion
- **Advanced metrics tracking** - Provides more precise session ID calculation and rotation by detecting user activity and idleness
```

we are calling this in multiple ways, lets stick to one name for this feature, the product seems to be 'session metrics'